tests/pamiq_core/console/test_cui.py

```diff
@@ -0,0 +1,212 @@
+import json
+import re
+import sys
+from collections.abc import Generator
+from unittest.mock import MagicMock, patch
```

Comment:
ここはMockerFixtureを使って、 `def test_case(mocker: MockerFixture)`として使う方式にpamiq-coreについては統一しているのでそうしてほしい（一部抜けもれはあるが...）
```suggestion
from pytest_mock import MockerFixture
```